Implementation of movie list

evvis opened the pull request:
Movie_list component creating. 
Movie service for API usage, encapsulating request. 
Output of the received data to the cards. Layout.

evvis committed:
Movie_list component creating. Movie service for API usage, encapsulating request. Output of the received data to the cards. Layout.

evvis committed:
Edit markup

evvis committed:
edit layout

maximprokopchuk commented on src/app/movie-list/movie-list.component.html:
```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div *ngFor="let movie of movie_list | async">
```
You should move this markup to separate component (you can name it, for example, `movie-card`)

maximprokopchuk commented on src/app/movie-list/movie-list.component.html:
```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div *ngFor="let movie of movie_list | async">
+      <div class="col s4 m2">
```
This markup is broken for different screen widths 
![image](https://user-images.githubusercontent.com/12671147/43575220-f4169ac2-964e-11e8-85a0-a35710dfe3ea.png) 
![image](https://user-images.githubusercontent.com/12671147/43575248-0a00aef4-964f-11e8-9d47-8ddb1c0ba4bf.png)

maximprokopchuk commented on src/app/services/movie.ts:
```diff
@@ -0,0 +1,43 @@
+export class Movie {
```
You use this class as interface (you don't create Movies instances `new Movie(...)`), it is needed just for typings, so you should replace this definition to interface definition

evvis committed:
Making changes according to the remarks.

evvis commented:
Renamed the "movie-list" component in "movieList"
A "movieCard" component has been created, the contents of "movieList" have been transferred to it.
Fixed layout.
The movie class is translated into the interface.

maximprokopchuk commented on src/app/movieList/movieCard/movieCard.component.html:
```diff
@@ -0,0 +1,29 @@
+<div class="row">
```
not  sure we need for rows for each card, bat leave it as it is now. will refactor later

maximprokopchuk commented on src/app/movieList/movieCard/movieCard.component.css:
```diff
@@ -0,0 +1,24 @@
+}
+
+.card {
+  height: 27.875em;
```
I said you should delete fixed height, it brokes markup on wide displays 
See - 
![image](https://user-images.githubusercontent.com/12671147/43592127-9312d400-967d-11e8-8d41-486c975c33e2.png)

maximprokopchuk reviewed (changes requested):
Rename files back. No need for camel case in filenames

evvis committed:
Making changes according to the remark

evvis commented:
Fixed.

evvis committed:
cleaning
